lib/storage/client.js

```diff
@@ -1044,6 +1047,65 @@ class Client extends CryptoConsumer {
     const params = { id_string: noteName }
     return shared.issueNoteChallenge(this, params, meta)
   }
+
+  /**
+   * Make a request object for the create group request
+   * @param {string} name
+   * @param {Array} capabilities
+   * @param {KeyPair} encryptionKeyPair
+   *
+   * @return {Group}
+   */
+  async createGroupRequest(name, capabilities = [], encryptionKeyPair) {
+    const groupKeyPair = await this.crypto.generateKeypair()
+    const eak = await this.crypto.encryptPrivateKey(
+      groupKeyPair.privateKey,
+      encryptionKeyPair.privateKey,
+      encryptionKeyPair.publicKey
+    )
+    let groupData = new GroupData(name, capabilities)
+    let membershipKeys = new GroupMembershipKeys(
+      encryptionKeyPair.publicKey,
+      eak
+    )
+    let group = new Group(groupData, null, membershipKeys)
```

Comment:
will call this out below in the type definitions, but it's a small code smell to see null passed to a constructor with any regularity. It generally means there is an improvement that can be made in the object composition.